packages/markdown-pdf/src/PdfTransformer.js

```diff
@@ -44,6 +46,28 @@ class PdfTransformer {
         return this.ciceroMarkTransformer.fromMarkdown(plainText, format);
 
     }
+
+    /**
+    * Convert a CiceroMark DOM to a pdf
+    * @param {*} input CiceroMark DOM
+    * @param {object} [options] Configuration options
+    * @param {string} [options.fileName] Name Of File
+    * @param {string} [options.path] path for output pdf
+    * @param {object} [options.config] markdown pdf Configuration
+    */
+
+    toPdf(input, options) {
+        options = options || {}
+        options.fileName = options.fileName ? '/' + options.fileName : '/accord.pdf'
+        options.path = options.path ? path.join(path.resolve(options.path), options.fileName) : path.join(process.cwd(), options.fileName)
+        options.config = options.config ? options.config : {}
+        //const ciceroMarkTransformer = new CiceroMarkTransformer();
+        const markdown = this.ciceroMarkTransformer.toMarkdown(input)
+        markdownpdf(options.config).from.string(markdown).to(options.path, function() {
```

Comment:
This callback makes this function asynchronous. How does a client know that the file has been written to disk?